src/main/java/org/schedulesdirect/api/NetworkEpgClient.java

```diff
@@ -423,6 +422,12 @@ public InputStream submitRequest(final DefaultJsonRequest req, final Object data
 	protected Program fetchProgram(final String progId) throws IOException {
 		return fetchPrograms(new String[] { progId }).values().toArray(new Program[1])[0];
 	}
+	
+	@Override
+	protected Artwork[] fetchArtwork(String progId) throws IOException {
```

Comment:
If fetching artwork isn't to be supported via the NetworkClient then this method should throw an UnsupportedOperationException so clients know that it will never return data.

In order for the ZipClient to have data to access, you'll have to modify the grabber to pull down and store this data into the zip file.